src/ussd_nurse.js

```diff
@@ -26,7 +26,7 @@ go.app = function() {
         var engage;
 
         self.init = function() {
-            var config = {headers: {'User-Agent': 'Jsbox/NDoH-Nurse'}};
+            var config = {headers: [{'User-Agent': 'Jsbox/NDoH-Nurse'}]};
```

Comment:
Is the list in the right place here?

Reply:
Ah, it's not, will fix

Reply:
Although we should probably remove this app, since we've moved over to RapidPro